data/json/snippets/mutant_anatomy.json

```diff
@@ -43,6 +43,182 @@
+      }
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "cooked_mutant_liver_desc",
+    "text": [
+      {
+        "id": "cooked_mutant_liver_1",
+        "text": "A cooked chunk of liver from a mutated animal.  Getting all the parasites off was more effort than this meal is probably worth, and the consistency suffered considerably from having to dig them out from the depth of the organ."
+      },
+      {
+        "id": "cooked_mutant_liver_2",
+        "text": "A cooked chunk of liver from a mutated animal.  Instead of the matte grey-brown of a normal liver, this one turned disconcertingly orange after heating it up, and finger-sized flakes detach themselves from it at the easiest touch.  The taste reminds you of slightly overripe avocado, but that's still better than you expected."
+      },
+      {
+        "id": "cooked_mutant_liver_3",
+        "text": "A cooked chunk of liver from a mutated animal.  The channels left by the hairs you removed fill with an oily black liquid, lending a surprisingly nice look to the cut surface.  Biting into it the fluid hardens unexpectedly, leaving you to chew much longer than you'd like to."
+      },
+      {
+        "id": "cooked_mutant_liver_4",
+        "text": "A cooked chunk of liver from a mutated animal.  The tissue split apart following the veins before hardening, reminding you of a warped grey flower with a sinous vein for a stem.  Surprisingly, it tastes not entirely unlike liver."
+      }
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "mutant_kidney_desc",
+    "text": [
+      {
+        "id": "mutant_kidney_1",
+        "text": "A piece of kidney from a mutated animal.  Cutting into it your knife slips, opening a large gash on its side from which tiny, beautiful pearls roll out.  Before your eyes they start glistening with humidity and melt, leaving no trace of them."
+      },
+      {
+        "id": "mutant_kidney_2",
+        "text": "A piece of kidney from a mutated animal.  The tissue is broken up by pointy, brittle crystals you need to carefully cut around, lest they embed in your hand.  When you're done about half the meat is gone."
+      },
+      {
+        "id": "mutant_kidney_3",
+        "text": "A piece of kidney from a mutated animal.  Most of the volume is taken up by clear cysts filled with liquid, leaving you to pop them one by one to try and salvage some usable tissue.  The result looks about as appetizing as it sounds, but you suppose it's technically edible."
+      },
+      {
+        "id": "mutant_kidney_4",
+        "text": "A piece of kidney from a mutated animal.  Accessory kidneys bud off the ureter in clumps, like red, slightly quivering grapes.  The main organ continues the same fractal structure, revealing itself to be a collection of smaller instances of each cut."
```

Comment:
"of each cut"?

Reply:
Getting there, but it shows that I did kidneys last.